python/main.py

```diff
@@ -18,17 +19,41 @@
     allow_headers=["*"],
 )
 
+items = list()
```

Comment:
it is usually a bad idea to hold the entire data in a global variable. Because there are multiple users updating this value at the same time, it may lead to Deadlocks. But that may also occur with `items.json` 

hence one of the reason we eventually use DB for storage.